bugtool/cmd/root.go

```diff
@@ -116,18 +116,18 @@ func init() {
 	)
 	BugtoolRootCmd.Flags().IntVar(&traceSeconds, "pprof-trace-seconds", 180, "Amount of seconds used for pprof CPU traces")
 	BugtoolRootCmd.Flags().StringVarP(&archiveType, "archiveType", "o", "tar", "Archive type: tar | gz")
-	BugtoolRootCmd.Flags().BoolVar(&k8s, "k8s-mode", false, "Require Kubernetes pods to be found or fail")
+	BugtoolRootCmd.Flags().BoolVar(&k8s, "k8s-mode", false, "Require Kubernetes pods to be found or fail (Deprecated)")
```

Comment:
Maybe use `MarkDeprecated` instead so the flag doesn't show up in usage information anymore and we don't need to manually extend the usage info:

```suggestion
	BugtoolRootCmd.Flags().BoolVar(&k8s, "k8s-mode", false, "Require Kubernetes pods to be found or fail")
	BugtoolRootCmd.Flags().MarkDeprecated("k8s-mode", "This option is deprecated, and will be removed in v1.18")
```

Same for the other flags below.

Reply:
TIL! Thanks for mentioning - changed.